### ISSUE TITLE
Analyse code and optimise as necessary

### ISSUE THREAD

**Issue by jodh-intel** [01org/cc-oci-runtime#307]
Once #271 is complete (and since new architectural elements are being added) we should measure, analyse and refine the code as an explicit step.

**Comment by dlespiau** [comment]
This task is a bit to general I think. We need to start with being able to produce consistent metrics. Then, we can look at defining exploration tasks based on those. How about changing that one to the first bit? just being able to gather metrics?

**Comment by jodh-intel** [comment]
Yes, rather a vague title, partly purposefully. I raised this specifically as a placeholder to remind us to compare 2.1 re-arch with the pre-2.1 re-arch version to ensure that not only have we not regressed metrics-wise, but also that we take the time to further optimise the new architecture once it's operational.

**Comment by jcvenegas** [comment]
@devimc can we close this one?

**Comment by jodh-intel** [comment]
I'm happy for us to close this, but what's youre view @grahamwhaley?

**Comment by grahamwhaley** [comment]
This has is indeed too general, and has been superseded by https://github.com/orgs/clearcontainers/projects/2 and all the Issues associated there, so I'm happy to close this one.
